rocky-backend/main.py

```
@@ -0,0 +1,122 @@
+    if not key:
+        return jsonify({"error": "API key not found"}), 404
+    key["_id"] = str(key["_id"])
+    return jsonify(key)
+
+@app.route("/api_keys/<key_id>", methods=["DELETE"])
+def delete_api_key(key_id):
+    api_keys.delete_one({"_id": ObjectId(key_id)})
+    return jsonify({"message": "API key deleted"})
+
+if __name__ == "__main__":
+    app.run(debug=True, port = 5001)
```

Ruff (0.15.6)

[error] Use of `debug=True` in Flask app detected

(S201)